learning_management_services/demo/sale_order_line.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_1" model="sale.order.line">
+        <field name="product_id" ref="product_product_3"/>
+        <field name="order_id" ref="sale_order_1"/>
+    </record>
+    <record id="sale_order_line_10" model="sale.order.line">
+        <field name="name">Discount: Free Product - All About Hotel management</field>
+        <field name="product_id" ref="product_product_22"/>
+        <field name="order_id" ref="sale_order_6"/>
+        <field name="reward_id" ref="loyalty_reward_4"/>
+        <field name="points_cost">5.0</field>
+    </record>
+    <record id="sale_order_line_11" model="sale.order.line">
+        <field name="name">Standard delivery</field>
+        <field name="product_id" ref="delivery.product_product_delivery"/>
+        <field name="order_id" ref="sale_order_6"/>
+        <field name="is_delivery" eval="True"/>
+    </record>
+    <record id="sale_order_line_12" model="sale.order.line">
+        <field name="product_id" ref="product_product_3"/>
+        <field name="order_id" ref="sale_order_7"/>
+    </record>
+    <record id="sale_order_line_13" model="sale.order.line">
+        <field name="name">Discount: Free Product - Become An Expert in Digital Marketing</field>
```

Comment:
I think we won't need the name to be explicitly added over here!
Try to remove it and see if it works properly in UI

Reply:
It is needed